src/delete-remote-gpu.py

```diff
@@ -141,14 +141,18 @@ async def _scorch_remote():
 
         print("⚠️ CAUTION, it'll remove the gpu from the remote inventory.\n")
 
-        selected_gpus = prompt_user_input(remote_inventory, ip_to_gpus)
-        if not selected_gpus:
-            return
+        if auto_delete:
+            selected_gpus = [inventory for inventory in orphaned_remote_gpus if inventory['chute'] is None and inventory['chute_id'] is None]
```

Comment:
I'm not 100% sure that the chute and chute_id values will be None in this case, but it's a good safeguard to add.